data/pizza.json

```diff
@@ -847,8 +847,8 @@
 				"hidden_ingredients": [
 					"tomatsås", "ost"
 				],
-				"price": 105,
-				"family_price": 230
+				"price": 110,
+				"family_price": 239
 			},
 			"rossi": {
```

Comment:
[It's the fårost again](https://sannegarden.com/meny/johanneberg/#:~:text=L%C3%B6k%2C%20F%C3%A4rsk%20Vitl%C3%B6k%2C%20Oliver%2C%20F%C3%A5rost%2C%20Lufttorkad%20Skinka%2C%20Olivolja%2C%20Basilika%2C%20Mozzarellaost)